### PR TITLE
Issue/4291 Align 'justified' in text gameobject

### PR DESCRIPTION
**Pull request by Donerkebap13**
Please do not update the README or Change Log, we will do this when we merge your PR.

This PR (delete as applicable)

* Adds a new feature

Describe the changes below:
Issue #4291 
Added the 'justified' align property to Text object

Usage:
this.add.text(100, 100, content, { fontFamily: 'Arial', color: '#00ff00', align: **'justified'** });

### CODE REVIEW

File: src/gameobjects/text/static/Text.js [hunk 1]
```diff
@@ -1198,6 +1198,27 @@ var Text = new Class({
             {
                 linePositionX += (textSize.width - textSize.lineWidths[i]) / 2;
             }
+            else if (style.align === 'justified')
+            {
+                var minimumLengthToApplyJustification = 0.85;
```
Comment (photonstorm) [inline src/gameobjects/text/static/Text.js]: Where does the 0.85 value come from?

Comment (dude78GH) [reply src/gameobjects/text/static/Text.js]: I guess this is some kind of ratio. I.e. if line width is less than 85% of defined width per line - there is no point of justifying it coz spaces would be too long.

**Comment by dude78GH** [comment]
Algorithm above needs few modifications I believe. Just compare.
This text uses code above to justify
Screenshot_2019-04-29_14-47-14

This one uses code below:
Screenshot_2019-04-29_14-47-50

```javascript
else if (style.align === 'justified')
{
    var minimumLengthToApplyJustification = 0.85;    // to justify text line its width must be not less than 85% of defined width
    if (textSize.lineWidths[i] / textSize.width >= minimumLengthToApplyJustification)
    {
        var extraSpace = textSize.width - textSize.lineWidths[i];
        var spaceSize = context.measureText(' ').width;
        var trimmedLine = lines[i].trim();
        var array = trimmedLine.split(' ');

        extraSpace += (lines[i].length - trimmedLine.length) * spaceSize;

        var extraSpaceCharacters = Math.floor(extraSpace / spaceSize);
        var idx = 0;
        while (extraSpaceCharacters > 0)
        {
            array[idx] += ' ';
            idx = (idx + 1) % (array.length - 1 || 1);
            --extraSpaceCharacters;
        }

        lines[i] = array.join(' ');
    }
}
```

**Comment by photonstorm** [comment]
Thanks for this PR (and the update to it @dude78GH). I have used the updated code and made the change locally, so although I won't merge this PR, the feature is still now in the master branch and you're both credited in the Change Log.

**Pull request closed without merging**